test/build_dependencie_graph.jl

```diff
@@ -0,0 +1,59 @@
+using IntegrationTests
+using PkgDependency
+using Pkg
+import Term.Trees: Tree
+
+# compare each node of both generated dependency graphs
+# the test requires, that both algorithm processes the dependencies in Pkg.project().dependencies()
+# and Pkg.dependencies()[uuid].dependencies() in the same order
+function compare_nodes(integTest, pkgDep, origin_string="graph: root")
+    if isnothing(integTest) || isnothing(pkgDep)
+        @test isnothing(integTest) == isnothing(pkgDep)
+        return nothing
+    end
+
+    integ_test_keys = collect(keys(integTest))
+    sort!(integ_test_keys)
+    # PkgDependency stores the package name together with the version number
+    # e.g. Term v1.0.1
+    # therefore remove version number, that packages can be compared
+    pkg_dep_keys = map(x -> split(x)[1], collect(keys(pkgDep)))
+    sort!(pkg_dep_keys)
+
+    @test length(integ_test_keys) == length(pkg_dep_keys)
+    @test integ_test_keys == pkg_dep_keys
+    # printing the whole graphs does provide useful debug information
+    # print more clever debug information
+    if length(integ_test_keys) != length(pkg_dep_keys) || integ_test_keys != pkg_dep_keys
+        println(origin_string)
+        println("IntegrationTests.build_dependencie_graph()\n$(integ_test_keys)")
```

Comment:
```suggestion
        println("IntegrationTests.build_dependency_graph()\n$(integ_test_keys)")
```
The file itself also still has the typo